### ISSUE TITLE
background and foreground treshold for removal

### ISSUE THREAD

**Issue by 3blackbar** [Loewen-Hob/rembg-comfyui-node-better#6]
Auto11 has that and helps a lot  with some images, you can control how much is treated as background and foreground.

**Comment by Loewen-Hob** [comment]
> Auto11 has that and helps a lot with some images, you can control how much is treated as background and foreground.

Yes yes, I was adjusting the thresholds in code before, I should have designed this as a visualisation that people can change by dragging, I'll make adjustments when I have time, thanks for the suggestion!
